pages/blog/_posts/ai-orchestration-with-agentkit-step-ai.mdx

```diff
@@ -0,0 +1,127 @@
+
+```ts
+import { Network, agenticOpenai } from "@inngest/agent-kit";
+import { navigator, classifier, summarizer } from "./src/agents";
+
+export default inngest.createFunction(
+  { id: "summarizer" },
+  { event: "api/summary.requested" },
+  async ({ event, step }) => {
+
+    // Create a network of agents with separate tasks and instructions to solve
+    // a specific task, using Inngest steps for orchestration 
+    const network = new Network({
+      agents: [navigator, classifier, summarizer],
+      defaultModel: agenticOpenai({ model: "gpt-4o", step }),
+    })
+    
+    const input = "Classify then summarize the latest 10 blog posts on https://www.deeplearning.ai/blog/"
+    
+    const result = await network.run(, ({ network }) => {
+      // Use an agent which figures out the specific agent to call based off of the network's history.
+      return defaultRoutingAgent;
+    });
+
+  },
+);
+```
+
+<a href="/docs/agent-kit/overview">You can read the docs here</a>, and [see the code on GitHub](https://github.com/inngest/agent-kit).  We cannot wait to see how you work with AgentKit.  We're iterating daily, and to speak with us or give feedback <a href="https://www.inngest.com/discord?ref=agentkit-blog"> join us in our #ai discord</a>
+
+
+## New step APIs - now available for early access: `step.ai.wrap` and `step.ai.infer`
+
+We’re launching two new step APIs:
+
+- `step.ai.wrap`, which wraps *any* AI SDK with reliable, retryable steps and full request/response AI observability
+- `step.ai.infer`, which securely offloads inference requests to any inference API through Inngest’s infrastructure, with full AI observability
+
+Both AI APIs offer a bunch of new features, specifically for rapidly iterating on production-ready API products:
+
+- Request and response tracking, in development and production
+- AI workflow tracing, including large sequences of inference calls or agentic workflows
+- The ability to edit inputs from the middle of a workflow and rerun
+- Metadata, metrics, and usage stats across AI model calls
+
+[To learn more, read our docs here](/docs/features/inngest-functions/steps-workflows/step-ai-orchestration).
+
+### `step.ai.wrap`: reliable AI with any SDK
+
+If you’ve already experimented with AI, you’re probably using an SDK like OpenAI’s, Anthropics, or Vercel’s AI SDK.  `step.ai.wrap` is designed to seamlessly wrap your existing AI SDK to with steps and AI observability:
+
+```typescript
+import { generateText } from "ai"
+import { openai } from "@ai-sdk/openai"
+
+export default inngest.createFunction(
+  { id: "summarize-contents" },
+  { event: "app/ticket.created" },
+  async ({ event, step }) => {
+  
+    // This calls `generateText` with the given arguments, adding AI observability,
+    // metrics, datasets, and monitoring to your calls.
+    const { text } = await step.ai.wrap("using-vercel-ai", generateText, {
+      model: openai("gpt-4-turbo"),
+      prompt: "What is love?"
+    });
+
+  }
+);
+```
```

Comment:
A screenshot of what you get for wrapping would be ideal - we're doing a bit of telling, not showing with the "with steps and AI observability" bit